.github/workflows/autorelease.yaml

```
@@ -0,0 +1,24 @@
+name: Autorelease
+on:
```

I would add   `workflow_dispatch: {}` to be able to run the action manually -> easier to test so you don't have to merge a commit into the branch (feel free to ignore)
```suggestion
on:
  workflow_dispatch: {} # workflow can be run manually
```

---

Amazing point. Will add that as well but it needs some docs